cmd/thanos/store.go

```diff
@@ -5,10 +5,13 @@ import (
 	"net"
 	"time"
 
+	"github.com/thanos-io/thanos/pkg/component"
+	"github.com/thanos-io/thanos/pkg/prober"
+
```

Comment:
you need to kill this line and run `make format` ):

Reply:
should be properly formatted now :crossed_fingers: